data/lib/src/network/linshare_http_client.dart

```diff
@@ -106,7 +109,27 @@ class LinShareHttpClient {
   }
 
   Future<List<SharedSpaceNodeNestedResponse>> getSharedSpaces() async {
-    final List resultJson = await _dioClient.get(Endpoint.sharedSpaces.withQueryParameters(['withRole=true']).generateEndpointPath());
+    final List resultJson = await _dioClient.get(Endpoint.sharedSpaces.withQueryParameters({'withRole': 'true'}).generateEndpointPath());
```

Comment:
If the map type is `Map<String, dynamic`, I think it could be `{'withRole': true}`